bin/p2-rctl/main.go

```diff
@@ -113,15 +115,24 @@ func main() {
 	httpClient := cleanhttp.DefaultClient()
 	client := consul.NewConsulClient(opts)
 
+	rcStore := rcstore.NewConsul(client, labeler, 3)
 	rctl := rctlParams{
 		httpClient: httpClient,
 		baseClient: client,
-		rcs:        rcstore.NewConsul(client, labeler, 3),
-		rls:        rollstore.NewConsul(client, labeler, nil),
-		consuls:    consul.NewConsulStore(client),
-		labeler:    labeler,
-		hcheck:     checker.NewConsulHealthChecker(client),
-		logger:     logger,
+		// rcStore copied so many times right now but might not all be
+		// the same implementation of these various interfaces in
+		// the future. Also rcWatcher isn't even used it is simply
+		// used by the roll farm to initialie an RC for a computation
```

Comment:
Oh. And you know, #792 removed that. So, at this point the only thing using rcWatcher is update_test.go I beieve.